src/app/components/coin/coin.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { DatePipe } from '@angular/common';
+
+@Component({
+  selector: 'app-coin',
+  templateUrl: './coin.component.html',
+  styleUrls: ['./coin.component.css']
+})
+export class CoinComponent implements OnInit {
+
+  @Input() coin: any;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  formatDate(date:string){
```

Comment:
Nu mai ai nevoie de aceasta metoda. Poti sa folosesti `| date: 'short'` in html :smile: